src/main/cluster.ts

```diff
@@ -169,6 +169,12 @@ export class Cluster implements ClusterModel, ClusterState {
    * @observable
    */
   @observable isAdmin = false;
+  /**
+   * Is cluster marked as dead, for example due the invalid kubeconfig
+   *
+   * @observable
+   */
+  @observable isDead = false;
```

Comment:
This doesn't need to be observable since it is only set in the constructor.

Reply:
changed as public property now.